src/intl/Core__Intl__NumberFormat.res

```diff
@@ -1,28 +1,235 @@
+module Grouping = Core__Intl__NumberFormat__Grouping
+
 type t
 
+/**
+An ISO 4217 currency code. e.g. USD, EUR, CNY
+*/
+type currency = string
+type currencyDisplay = [#symbol | #narrowSymbol | #code | #name]
+type currencySign = [#accounting | #standard]
+type notation = [#scientific | #standard | #engineering | #compact]
+
+/**
+Used only when notation is #compact
+*/
+type compactDisplay = [#short | #long]
+
+// TODO: "negative" is not yet well supported
```

Comment:
Needs a decision on to include this or not

Reply:
IMHO we should include it, even if not all JS runtimes support it (yet).

Reply:
@illusionalsagacity Just noticed that this is also still open. Let's include it. Maybe add a doc comment stating that it is not supported in all runtimes yet?

Reply:
Looks like we are good on compatibility now for this anyway